### DISCUSSION TITLE
Skill points on humanoids

### DISCUSSION THREAD

**Discussion by LitroTheDragon** [League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima#191] [General]
I noticed the "Total SP used" under a sheets settings still counts skills from equipment even if the NPC is humanoid, even though humanoids gain the **Use Equipment** skill for free. Is this a mistake? Or did i misread something when putting NPCs together?

**Comment by spyrella** [comment]
That is in fact a typo. Will try to get that handled by next patch.
